iree/compiler/Dialect/VMLA/Transforms/PreConversionLowering.cpp

```diff
@@ -277,6 +277,95 @@ class LowerBroadcastOp : public OpRewritePattern<mhlo::BroadcastOp> {
   }
 };
 
+// Lower mhlo::SortOp to an pseudo SortOp in the VMLA dialect. This
+// pseudo op generates a set of ordered indices for that array along the last
+// dimension. Then using a torch_index_select the values can be reordered to
+// support arbitrary inputs.
+//
+// Note: At this point only organizes ascending values for a single input.
+// TODO(suderman): support descending support.
```

Comment:
Can you elaborate on this, perhaps with samples of what is/isn't supported?

Reply:
Updated with a more explicit explanation (specifically ascending vs descending operations).